### ISSUE TITLE
Enhancement: Allow overwrite of the `operation_id` value

### ISSUE THREAD

**Issue by WardPearce** [starlite-api/starlite#1132]
**What's the feature you'd like to ask for.**
When defining routes, specifying `operation_id` doesn't overwrite the full `operationId`.
For example
```py
@get(
    path="/kdf/{username:str}",
    description="Public KDF details for user",
    tags=["user"],
    raises=[UserNotFound],
    middleware=[RateLimitConfig(rate_limit=("minute", 3)).middleware],
    summary="Kdf",
    operation_id="Kdf",
)
async def kdf(username: str) -> Argon2Modal:
    user = await User(username=username).get()
    return user.kdf
```
results in the `"operationId": "ControllerUserKdfKdf"`

Ideally I'd like to be able to set the `operationId` as  `ControllerUserKdf`. Setting `operation_id=""`, still results in `"operationId": "ControllerUserKdfKdf"`

**Additional context**
Not having this control over the `operationId` has resulted in ugly generated API clients for the frontend code. What makes it less developer friendly.

**Comment by WardPearce** [comment]
> @WardPearce - PR added for this. Its going to be included in v2.0

Awesome! looking forward to it. Was thinking being able to control the pattern for how operation_id is made could be useful too. E.g. `{route}_{operation_id}`

**Comment by Goldziher** [comment]
> > @WardPearce - PR added for this. Its going to be included in v2.0
> 
> Awesome! looking forward to it. Was thinking being able to control the pattern for how operation_id is made could be useful too. E.g. `{route}_{operation_id}`

That's included in that OR too
